testSchema.sh

```diff
@@ -6,11 +6,11 @@ set -e
 declare -a testCases
 testCases=(
 bpa "data.bloodpac.org"
-bhc "data.braincommons.org"
-gtex "dcp.bionimbus.org"
+gtex "gen3.biodatacatalyst.nhlbi.nih.gov"
```

Comment:
Just confirming that this `gtex` change was intentional 👀 while removing the `bhc` stuff

Reply:
Yeah I feel we may need it. Because we've changed gtex to bdcat? If not I'll just change it back 😅

Reply:
👍